initiatives/info.yml

```diff
@@ -0,0 +1,57 @@
+---
+resource: budgets
+query:
+  model: true
+  sql: |
+    select 
+        decidim_initiatives.id,
+        decidim_initiatives.id::text as id_as_text,
+        title->>'fr' as title,
+        regexp_replace(decidim_initiatives.description->>'fr', E'(<[^>]+>)|(&[a-z]+;)', '', 'gi') as description,
```

Comment:
same as above, switch `fr` to `$LANGUAGE_CODE`

Reply:
fixed in [484818f](https://github.com/OpenSourcePolitics/decidim_cards/pull/65/commits/484818f4205f309fd8a3d8ff5282730828d94bb1)